docs/index.html

```diff
@@ -150,164 +150,165 @@ <h1 class=".page-header">Available Items</h1>
       <div class="grid grid-container" id="mainGrid">
 
         <div class="grid-sizer"></div>
-            <div class="grid-item">
-                <a href="#">
-                    <div class="thumbnail">
-
-                        <img src="images/couch.jpg" class="img img-thumbnail"/>
-                    </div>
-                </a>
-            </div>
+          
+        <div id="couch.jpg" class="grid-item">
```

Comment:
They won't be hardcoded. They will be added in when the page loads. I just have these here so that I can look at the layout and stuff until I figure out where the images are going to live and how to load them onto the page.